src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
 
     @Before
```

Comment:
If this "setUp" is intended to be run once before any @Tests, then it should be annotated with @BeforeClass... and therefore be static.